packages/core/js/colors.2020.js

```diff
@@ -0,0 +1,151 @@
+export const blue = {
+  base: '#0084BD',
+  1: '#D9F1FF',
+  2: '#B9E4FD',
+  3: '#8ED1F6',
+  4: '#58B9EB',
+  5: '#2EA0D6',
+  6: '#0084BD',
+  7: '#0074AB',
+  8: '#006395',
+  9: '#005685',
+  10: '#00446B'
+}
+export const teal = {
+  base: '#00BAA5',
+  1: '#D5FAF5',
+  2: '#AFFAEF',
+  3: '#84F0E1',
+  4: '#5CE6D4',
+  5: '#2FD1BE',
+  6: '#00BAA5',
+  7: '#00A894',
+  8: '#009380',
+  9: '#007A6A',
+  10: '#006658'
+}
+export const green = {
+  base: '#009E52',
+  1: '#D9FAE5',
+  2: '#ADF0C8',
+  3: '#82E2AB',
+  4: '#57D08E',
+  5: '#2BB970',
+  6: '#009E52',
+  7: '#008F46',
+  8: '#007C3A',
+  9: '#00672E',
+  10: '#005724'
+}
+export const lime = {
+  base: '#8CCA08',
+  1: '#ECFFC7',
+  2: '#DEFFA3',
+  3: '#CCFC7A',
+  4: '#B8F052',
+  5: '#A3E029',
+  6: '#8CCA08',
+  7: '#7BB600',
+  8: '#6BA300',
+  9: '#588A00',
+  10: '#4C7700'
+}
+export const yellow = {
+  base: '#FAD000',
+  1: '#FFFBCC',
+  2: '#FFF7A8',
+  3: '#FFF380',
+  4: '#FFEB57',
+  5: '#FFDF29',
+  6: '#FAD000',
+  7: '#E2B500',
+  8: '#CC9E00',
+  9: '#B28300',
+  10: '#946500'
+}
+export const orange = {
+  base: '#F59127',
+  1: '#FFECD1',
+  2: '#FFDCAD',
+  3: '#FFCD8A',
+  4: '#FFBA61',
+  5: '#FFA43E',
+  6: '#F59127',
+  7: '#E27A18',
+  8: '#CB670B',
+  9: '#B85500',
+  10: '#9E4100'
+}
+export const red = {
+  base: '#D21C09',
+  1: '#FFD6D6',
+  2: '#FFB3B3',
+  3: '#FF8A8A',
+  4: '#F86968',
+  5: '#E94A3F',
+  6: '#D21C09',
+  7: '#C00F00',
+  8: '#AB0600',
+  9: '#920000',
+  10: '#750000'
+}
+export const pink = {
+  base: '#E7558B',
+  1: '#FFDDEB',
+  2: '#FFC2DB',
+  3: '#FFA3C8',
+  4: '#FF8AB5',
+  5: '#F66FA1',
+  6: '#E7558B',
+  7: '#D1487B',
+  8: '#C13368',
+  9: '#A22554',
+  10: '#8A1A45'
+}
+export const purple = {
+  base: '#8359DF',
+  1: '#E9DEFF',
+  2: '#D6C2FF',
+  3: '#C0A3FF',
+  4: '#A883F8',
+  5: '#956CED',
+  6: '#8359DF',
+  7: '#7048C6',
+  8: '#5934AC',
+  9: '#482592',
+  10: '#351973'
+}
+
+export const colorsTextIcon = {
```

Comment:
wondering why these variables have a `colors` prefix but the vars above do not.

what if you define/export the variables in this file without a prefix and then in the `packages/core/js/index.js` you can rename the exported object like we do the other colors file

`export { default as colors2020 } from './colors.2020.js'`

I coming in a little late to the feature here. just thought this might be a better pattern

Reply:
They are tokens not just variables and this is to simply help identified them. Also should we be making an effort to reduce the size of bundles by using tree shaking patterns?

Reply:
Maybe it should be colorBlue instead of blue for example??

Reply:
Or maybe the `colorsTextIcon` is just `textIcon`?

Reply:
(use the abstraction of the package name for the "color" categorization)